search/routes/api/atom_extensions.py

```diff
@@ -0,0 +1,280 @@
+"""Classes derived from the Feedgen extension classes."""
+
+from typing import Any, Dict
+from feedgen.ext.base import BaseEntryExtension, BaseExtension
+from feedgen.entry import FeedEntry
+from feedgen.feed import FeedGenerator
+from lxml import etree
+
```

Comment:
```suggestion
ARXIV_NS = 'http://arxiv.org/schemas/atom'
OS_NS = 'http://a9.com/-/spec/opensearch/1.1/'

```
Consider defining these once at the top

Reply:
Fixed.